src/test/java/fr/spoonlabs/flacoco/api/FlacocoTest.java

```diff
@@ -1329,6 +1776,11 @@ public void testExampleCL2SpectrumBasedOchiaiDefaultMode() {
 		// Run default mode
 		FlacocoResult result = flacoco.run();
 
+		// TODO: Flacoco is running 743 tests compared to the baseline obtained through running `mvn clean test` on the cl2 project
+		// TODO: It also finds 4 failing tests, when it should be 3
+		// assertEquals(703, result.getExecutedTests().size());
+		//assertEquals(4, result.getFailingTests().size());
```

Comment:
Interestingly, we fail to run the same set of tests as `mvn test` for both real bugs we have under our test suite.

Reply:
Do you have the list of those cases we report as executed?

Reply:
I will look into the diff later today

Reply:
Baseline, obtained via `ag --skip-vcs-ignores --xml "<testcase classname" | cut -d' ' -f 4,5 | awk '{ print $1 "" $2}' | sed 's/classname="//' | sed 's/name="/#/' | sed 's/"//' | sed 's/"//'` on the surefire-report dir:
```
org.apache.commons.lang.util.IdentifierUtilsTest#testLongIncrementingNoArgs
org.apache.commons.lang.util.IdentifierUtilsTest#testLongIncrementingInit
org.apache.commons.lang.util.IdentifierUtilsTest#testLongIncrementingWrap
org.apache.commons.lang.util.IdentifierUtilsTest#testLongIncrementingNoWrap
org.apache.commons.lang.util.IdentifierUtilsTest#testStringNumericLong
org.apache.commons.lang.util.IdentifierUtilsTest#testStringNumericNoArgs
org.apache.commons.lang.util.IdentifierUtilsTest#testStringNumericInit
org.apache.commons.lang.util.IdentifierUtilsTest#testStringNumericWrap
org.apache.commons.lang.util.IdentifierUtilsTest#testStringNumericNoWrap
org.apache.commons.lang.util.IdentifierUtilsTest#testStringAlphanumeric
org.apache.commons.lang.util.IdentifierUtilsTest#testLongAlphanumericNoArgs
org.apache.commons.lang.util.IdentifierUtilsTest#testStringAlphanumericWrap
org.apache.commons.lang.util.IdentifierUtilsTest#testStringAlphanumericNoWrap
org.apache.commons.lang.util.IdentifierUtilsTest#testStringSession
org.apache.commons.lang.util.IdentifierUtilsTest#testLongIncrementing
org.apache.commons.lang.time.StopWatchTest#testStopWatchSimple
org.apache.commons.lang.time.StopWatchTest#testStopWatchSplit
org.apache.commons.lang.time.StopWatchTest#testStopWatchSuspend
org.apache.commons.lang.enums.EnumUtilsTest#testMap
org.apache.commons.lang.enums.EnumUtilsTest#testGetValue
org.apache.commons.lang.enums.EnumUtilsTest#testGet
org.apache.commons.lang.enums.EnumUtilsTest#testIterator
org.apache.commons.lang.enums.EnumUtilsTest#testListEx
org.apache.commons.lang.enums.EnumUtilsTest#testMapEx
org.apache.commons.lang.enums.EnumUtilsTest#testIteratorEx
org.apache.commons.lang.enums.EnumUtilsTest#testGetEx
org.apache.commons.lang.enums.EnumUtilsTest#testGetValueEx
org.apache.commons.lang.enums.EnumUtilsTest#testList
org.apache.commons.lang.IllegalClassExceptionTest#testGetMessage_stringArg_nullInput
org.apache.commons.lang.IllegalClassExceptionTest#testGetMessage_stringArg_validInput
org.apache.commons.lang.IllegalClassExceptionTest#testConstructor_stringArg_nullInput
org.apache.commons.lang.IllegalClassExceptionTest#testConstructor_classArgs_allNullInput
org.apache.commons.lang.IllegalClassExceptionTest#testConstructor_classArgs_nullExpected
org.apache.commons.lang.IllegalClassExceptionTest#testConstructor_classArgs_nullActual
org.apache.commons.lang.IllegalClassExceptionTest#testGetMessage_classArgs_nullInput
org.apache.commons.lang.IllegalClassExceptionTest#testGetMessage_classArgs_normalInput
org.apache.commons.lang.time.DurationFormatUtilsTest#testFormatWords
org.apache.commons.lang.time.DurationFormatUtilsTest#testFormatISOStyle
org.apache.commons.lang.builder.HashCodeBuilderTest#testByte
org.apache.commons.lang.builder.HashCodeBuilderTest#testConstructorEx1
org.apache.commons.lang.builder.HashCodeBuilderTest#testConstructorEx2
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHashCode
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHierarchyHashCode
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHierarchyHashCodeEx1
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHierarchyHashCodeEx2
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHashCodeEx1
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHashCodeEx2
org.apache.commons.lang.builder.HashCodeBuilderTest#testReflectionHashCodeEx3
org.apache.commons.lang.builder.HashCodeBuilderTest#testObjectArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testLongArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testIntArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testShortArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testCharArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testByteArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testDoubleArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testFloatArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testBooleanArrayAsObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testBooleanMultiArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testObject
org.apache.commons.lang.builder.HashCodeBuilderTest#testLong
org.apache.commons.lang.builder.HashCodeBuilderTest#testInt
org.apache.commons.lang.builder.HashCodeBuilderTest#testShort
org.apache.commons.lang.builder.HashCodeBuilderTest#testChar
org.apache.commons.lang.builder.HashCodeBuilderTest#testDouble
org.apache.commons.lang.builder.HashCodeBuilderTest#testFloat
org.apache.commons.lang.builder.HashCodeBuilderTest#testBoolean
org.apache.commons.lang.builder.HashCodeBuilderTest#testObjectArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testLongArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testIntArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testShortArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testCharArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testByteArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testDoubleArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testFloatArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testBooleanArray
org.apache.commons.lang.builder.HashCodeBuilderTest#testSuper
org.apache.commons.lang.util.ValidateTest#testIsTrue2
org.apache.commons.lang.util.ValidateTest#testIsTrue3
org.apache.commons.lang.util.ValidateTest#testIsTrue1
org.apache.commons.lang.util.ValidateTest#testIsTrue4
org.apache.commons.lang.util.ValidateTest#testIsTrue5
org.apache.commons.lang.util.ValidateTest#testNotNull1
org.apache.commons.lang.util.ValidateTest#testNotNull2
org.apache.commons.lang.util.ValidateTest#testNotEmptyArray1
org.apache.commons.lang.util.ValidateTest#testNotEmptyArray2
org.apache.commons.lang.util.ValidateTest#testNotEmptyCollection1
org.apache.commons.lang.util.ValidateTest#testNotEmptyCollection2
org.apache.commons.lang.util.ValidateTest#testNotEmptyMap1
org.apache.commons.lang.util.ValidateTest#testNotEmptyMap2
org.apache.commons.lang.util.ValidateTest#testNotEmptyString1
org.apache.commons.lang.util.ValidateTest#testNotEmptyString2
org.apache.commons.lang.util.ValidateTest#testNoNullElementsArray1
org.apache.commons.lang.util.ValidateTest#testNoNullElementsArray2
org.apache.commons.lang.util.ValidateTest#testNoNullElementsCollection1
org.apache.commons.lang.util.ValidateTest#testNoNullElementsCollection2
org.apache.commons.lang.time.DateFormatUtilsTest#testTimeNoTISO
org.apache.commons.lang.time.DateFormatUtilsTest#testDateTimeISO
org.apache.commons.lang.time.DateFormatUtilsTest#testSMTP
org.apache.commons.lang.time.DateFormatUtilsTest#testDateISO
org.apache.commons.lang.time.DateFormatUtilsTest#testTimeISO
org.apache.commons.lang.StringUtilsTest#testChopNewLine_8
org.apache.commons.lang.StringUtilsSubstringTest#testLeft
org.apache.commons.lang.StringUtilsSubstringTest#testLeftEx
org.apache.commons.lang.StringUtilsSubstringTest#testSubstring3
org.apache.commons.lang.StringUtilsSubstringTest#testSubstring2
org.apache.commons.lang.StringUtilsSubstringTest#testRight
org.apache.commons.lang.StringUtilsSubstringTest#testRightEx
org.apache.commons.lang.StringUtilsSubstringTest#testMid
org.apache.commons.lang.StringUtilsSubstringTest#testMidEx1
org.apache.commons.lang.StringUtilsSubstringTest#testMidEx2
org.apache.commons.lang.StringUtilsSubstringTest#testMidEx3
org.apache.commons.lang.StringUtilsSubstringTest#testCountMatches
org.apache.commons.lang.StringUtilsSubstringTest#testGetNestedString
org.apache.commons.lang.UnhandledExceptionTest#testConstructor_throwable_nullInput
org.apache.commons.lang.UnhandledExceptionTest#testConstructor_stringAndThrowable_nullInput
org.apache.commons.lang.UnhandledExceptionTest#testGetCauseAndGetMessage
org.apache.commons.lang.UnhandledExceptionTest#testGetCause
org.apache.commons.lang.math.RandomUtilsTest#testNextDouble
org.apache.commons.lang.math.RandomUtilsTest#testNextDouble2
org.apache.commons.lang.math.RandomUtilsTest#testSetSeed
org.apache.commons.lang.math.RandomUtilsTest#testNextInt
org.apache.commons.lang.math.RandomUtilsTest#testNextInt2
org.apache.commons.lang.math.RandomUtilsTest#testNextLong
org.apache.commons.lang.math.RandomUtilsTest#testNextLong2
org.apache.commons.lang.math.RandomUtilsTest#testNextBoolean
org.apache.commons.lang.math.RandomUtilsTest#testNextBoolean2
org.apache.commons.lang.math.RandomUtilsTest#testNextFloat
org.apache.commons.lang.math.RandomUtilsTest#testNextFloat2
org.apache.commons.lang.builder.DefaultToStringStyleTest#testLongArrayArray
org.apache.commons.lang.builder.DefaultToStringStyleTest#testAppendSuper
org.apache.commons.lang.builder.DefaultToStringStyleTest#testObject
org.apache.commons.lang.builder.DefaultToStringStyleTest#testLong
org.apache.commons.lang.builder.DefaultToStringStyleTest#testObjectArray
org.apache.commons.lang.builder.DefaultToStringStyleTest#testLongArray
org.apache.commons.lang.builder.DefaultToStringStyleTest#testBlank
org.apache.commons.lang.SystemUtilsTest#testJavaVersionMatches
org.apache.commons.lang.SystemUtilsTest#testJavaVersionAsFloat
org.apache.commons.lang.SystemUtilsTest#testJavaVersionAsInt
org.apache.commons.lang.SystemUtilsTest#testJavaVersionAtLeastFloat
org.apache.commons.lang.SystemUtilsTest#testJavaVersionAtLeastInt
org.apache.commons.lang.SystemUtilsTest#testOSMatches
org.apache.commons.lang.SystemUtilsTest#testOSMatches2
org.apache.commons.lang.StringUtilsIsTest#testIsAlphanumeric
org.apache.commons.lang.StringUtilsIsTest#testIsAlphanumericSpace
org.apache.commons.lang.StringUtilsIsTest#testIsNumeric
org.apache.commons.lang.StringUtilsIsTest#testIsNumericSpace
org.apache.commons.lang.StringUtilsIsTest#testIsAlpha
org.apache.commons.lang.StringUtilsIsTest#testIsAlphaspace
org.apache.commons.lang.StringUtilsIsTest#testIsWhitespace
org.apache.commons.lang.enums.ValuedEnumTest#testName
org.apache.commons.lang.enums.ValuedEnumTest#testEquals
org.apache.commons.lang.enums.ValuedEnumTest#testToString
org.apache.commons.lang.enums.ValuedEnumTest#testMap
org.apache.commons.lang.enums.ValuedEnumTest#testGetValue
org.apache.commons.lang.enums.ValuedEnumTest#testGet
org.apache.commons.lang.enums.ValuedEnumTest#testSerialization
org.apache.commons.lang.enums.ValuedEnumTest#testCompareTo
org.apache.commons.lang.enums.ValuedEnumTest#testIterator
org.apache.commons.lang.enums.ValuedEnumTest#testList
org.apache.commons.lang.enums.ValuedEnumTest#testValue
org.apache.commons.lang.StringEscapeUtilsTest#testUnescapeJava
org.apache.commons.lang.StringEscapeUtilsTest#testEscapeJavaScript
org.apache.commons.lang.StringEscapeUtilsTest#testEscapeHtml
org.apache.commons.lang.StringEscapeUtilsTest#testUnescapeHtml
org.apache.commons.lang.StringEscapeUtilsTest#testEscapeJava
org.apache.commons.lang.StringEscapeUtilsTest#testUnescapeUnknownEntity
org.apache.commons.lang.StringEscapeUtilsTest#testEscapeHtmlVersions
org.apache.commons.lang.StringEscapeUtilsTest#testEscapeXml
org.apache.commons.lang.ArrayUtilsTest#testContains
org.apache.commons.lang.ArrayUtilsTest#testReverseDouble
org.apache.commons.lang.ArrayUtilsTest#testSameLengthShort
org.apache.commons.lang.ArrayUtilsTest#testSameLengthByte
org.apache.commons.lang.ArrayUtilsTest#testSameLengthDouble
org.apache.commons.lang.ArrayUtilsTest#testSameLengthChar
org.apache.commons.lang.ArrayUtilsTest#testSameLengthFloat
org.apache.commons.lang.ArrayUtilsTest#testToString
org.apache.commons.lang.ArrayUtilsTest#testIndexOfWithStartIndex
org.apache.commons.lang.ArrayUtilsTest#testLastIndexOfWithStartIndex
org.apache.commons.lang.ArrayUtilsTest#testClone
org.apache.commons.lang.ArrayUtilsTest#testSameLengthBoolean
org.apache.commons.lang.ArrayUtilsTest#testHashCode
org.apache.commons.lang.ArrayUtilsTest#testCloneBoolean
org.apache.commons.lang.ArrayUtilsTest#testCloneLong
org.apache.commons.lang.ArrayUtilsTest#testCloneInt
org.apache.commons.lang.ArrayUtilsTest#testCloneShort
org.apache.commons.lang.ArrayUtilsTest#testCloneChar
org.apache.commons.lang.ArrayUtilsTest#testCloneByte
org.apache.commons.lang.ArrayUtilsTest#testCloneDouble
org.apache.commons.lang.ArrayUtilsTest#testCloneFloat
org.apache.commons.lang.ArrayUtilsTest#testSameLength
org.apache.commons.lang.ArrayUtilsTest#testSameType
org.apache.commons.lang.ArrayUtilsTest#testReverse
org.apache.commons.lang.ArrayUtilsTest#testReverseLong
org.apache.commons.lang.ArrayUtilsTest#testReverseInt
org.apache.commons.lang.ArrayUtilsTest#testReverseShort
org.apache.commons.lang.ArrayUtilsTest#testReverseChar
org.apache.commons.lang.ArrayUtilsTest#testReverseByte
org.apache.commons.lang.ArrayUtilsTest#testReverseFloat
org.apache.commons.lang.ArrayUtilsTest#testReverseBoolean
org.apache.commons.lang.ArrayUtilsTest#testIndexOf
org.apache.commons.lang.ArrayUtilsTest#testLastIndexOf
org.apache.commons.lang.ArrayUtilsTest#testToMap
org.apache.commons.lang.ArrayUtilsTest#testIsEquals
org.apache.commons.lang.ArrayUtilsTest#testSameLengthLong
org.apache.commons.lang.ArrayUtilsTest#testSameLengthInt
org.apache.commons.lang.builder.SimpleToStringStyleTest#testLongArrayArray
org.apache.commons.lang.builder.SimpleToStringStyleTest#testAppendSuper
org.apache.commons.lang.builder.SimpleToStringStyleTest#testObject
org.apache.commons.lang.builder.SimpleToStringStyleTest#testLong
org.apache.commons.lang.builder.SimpleToStringStyleTest#testObjectArray
org.apache.commons.lang.builder.SimpleToStringStyleTest#testLongArray
org.apache.commons.lang.builder.SimpleToStringStyleTest#testBlank
org.apache.commons.lang.NumberUtilsTest#testStringToIntString
org.apache.commons.lang.NumberUtilsTest#testStringToIntStringI
org.apache.commons.lang.NumberUtilsTest#testCreateInteger
org.apache.commons.lang.NumberUtilsTest#testCreateLong
org.apache.commons.lang.NumberUtilsTest#testCreateBigInteger
org.apache.commons.lang.NumberUtilsTest#testCreateBigDecimal
org.apache.commons.lang.NumberUtilsTest#testMinimumLong
org.apache.commons.lang.NumberUtilsTest#testMinimumInt
org.apache.commons.lang.NumberUtilsTest#testMaximumLong
org.apache.commons.lang.NumberUtilsTest#testMaximumInt
org.apache.commons.lang.NumberUtilsTest#testCompareDouble
org.apache.commons.lang.NumberUtilsTest#testCompareFloat
org.apache.commons.lang.NumberUtilsTest#testIsDigits
org.apache.commons.lang.NumberUtilsTest#testIsNumber
org.apache.commons.lang.NumberUtilsTest#testConstants
org.apache.commons.lang.NumberUtilsTest#testCreateFloat
org.apache.commons.lang.NumberUtilsTest#testCreateNumber
org.apache.commons.lang.NumberUtilsTest#testCreateDouble
org.apache.commons.lang.CharSetUtilsTest#testSqueeze
org.apache.commons.lang.CharSetUtilsTest#testKeep
org.apache.commons.lang.CharSetUtilsTest#testCount
org.apache.commons.lang.CharSetUtilsTest#testDelete
org.apache.commons.lang.NotImplementedExceptionTest#testConstructor_classArg_nullInput
org.apache.commons.lang.NotImplementedExceptionTest#testGetMessage_stringArg_nullInput
org.apache.commons.lang.NotImplementedExceptionTest#testGetMessage_stringArg_validInput
org.apache.commons.lang.NotImplementedExceptionTest#testConstructor_stringArg_nullInput
org.apache.commons.lang.NotImplementedExceptionTest#testGetMessage_classArg_nullInput
org.apache.commons.lang.NotImplementedExceptionTest#testGetMessage_classArg_validInput
org.apache.commons.lang.time.FastDateFormatTest#test_getInstance
org.apache.commons.lang.time.FastDateFormatTest#test_getInstance_String_TimeZone
org.apache.commons.lang.time.FastDateFormatTest#test_getInstance_String_Locale
org.apache.commons.lang.time.FastDateFormatTest#test_getInstance_String_TimeZone_Locale
org.apache.commons.lang.time.FastDateFormatTest#testFormat
org.apache.commons.lang.time.FastDateFormatTest#test_getInstance_String
org.apache.commons.lang.math.NumberRangeTest#testIncludesNumber
org.apache.commons.lang.math.NumberRangeTest#testConstructor2
org.apache.commons.lang.math.NumberRangeTest#testConstructor1
org.apache.commons.lang.math.NumberRangeTest#testIncludesLongBig
org.apache.commons.lang.math.NumberRangeTest#testEquals
org.apache.commons.lang.math.NumberRangeTest#testIncludesRange
org.apache.commons.lang.math.NumberRangeTest#testToString
org.apache.commons.lang.math.NumberRangeTest#testIncludesInteger
org.apache.commons.lang.math.NumberRangeTest#testGetMinimum
org.apache.commons.lang.math.NumberRangeTest#testGetMaximum
org.apache.commons.lang.math.NumberRangeTest#testIncludesLong
org.apache.commons.lang.math.NumberRangeTest#testIncludesDouble
org.apache.commons.lang.math.NumberRangeTest#testIncludesFloat
org.apache.commons.lang.math.NumberRangeTest#testOverlapsRange
org.apache.commons.lang.math.NumberRangeTest#testHashCode
org.apache.commons.lang.math.FractionTest#testEquals
org.apache.commons.lang.math.FractionTest#testToString
org.apache.commons.lang.math.FractionTest#testConstants
org.apache.commons.lang.math.FractionTest#testFactory_int_int
org.apache.commons.lang.math.FractionTest#testFactory_int_int_int
org.apache.commons.lang.math.FractionTest#testReducedFactory_int_int
org.apache.commons.lang.math.FractionTest#testFactory_double
org.apache.commons.lang.math.FractionTest#testFactory_String
org.apache.commons.lang.math.FractionTest#testFactory_String_double
org.apache.commons.lang.math.FractionTest#testFactory_String_proper
org.apache.commons.lang.math.FractionTest#testFactory_String_improper
org.apache.commons.lang.math.FractionTest#testGets
org.apache.commons.lang.math.FractionTest#testConversions
org.apache.commons.lang.math.FractionTest#testReduce
org.apache.commons.lang.math.FractionTest#testInvert
org.apache.commons.lang.math.FractionTest#testNegate
org.apache.commons.lang.math.FractionTest#testAbs
org.apache.commons.lang.math.FractionTest#testPow
org.apache.commons.lang.math.FractionTest#testAdd
org.apache.commons.lang.math.FractionTest#testSubtract
org.apache.commons.lang.math.FractionTest#testMultiply
org.apache.commons.lang.math.FractionTest#testDivide
org.apache.commons.lang.math.FractionTest#testToProperString
org.apache.commons.lang.math.FractionTest#testCompareTo
org.apache.commons.lang.math.FractionTest#testHashCode
org.apache.commons.lang.enums.EnumTest#testName
org.apache.commons.lang.enums.EnumTest#testEquals
org.apache.commons.lang.enums.EnumTest#testToString
org.apache.commons.lang.enums.EnumTest#testMap
org.apache.commons.lang.enums.EnumTest#testGet
org.apache.commons.lang.enums.EnumTest#testSerialization
org.apache.commons.lang.enums.EnumTest#testCompareTo
org.apache.commons.lang.enums.EnumTest#testIterator
org.apache.commons.lang.enums.EnumTest#testOperationGet
org.apache.commons.lang.enums.EnumTest#testOperationSerialization
org.apache.commons.lang.enums.EnumTest#testOperationToString
org.apache.commons.lang.enums.EnumTest#testOperationList
org.apache.commons.lang.enums.EnumTest#testOperationMap
org.apache.commons.lang.enums.EnumTest#testOperationCalculation
org.apache.commons.lang.enums.EnumTest#testExtended1Get
org.apache.commons.lang.enums.EnumTest#testExtended2Get
org.apache.commons.lang.enums.EnumTest#testExtended3Get
org.apache.commons.lang.enums.EnumTest#testExtendedSerialization
org.apache.commons.lang.enums.EnumTest#testExtendedToString
org.apache.commons.lang.enums.EnumTest#testExtended1List
org.apache.commons.lang.enums.EnumTest#testExtended2List
org.apache.commons.lang.enums.EnumTest#testExtended3List
org.apache.commons.lang.enums.EnumTest#testExtended1Map
org.apache.commons.lang.enums.EnumTest#testExtended2Map
org.apache.commons.lang.enums.EnumTest#testExtended3Map
org.apache.commons.lang.enums.EnumTest#testBroken2
org.apache.commons.lang.enums.EnumTest#testBroken3
org.apache.commons.lang.enums.EnumTest#testBroken1
org.apache.commons.lang.enums.EnumTest#testList
org.apache.commons.lang.ClassUtilsTest#test_getShortClassName_Object
org.apache.commons.lang.ClassUtilsTest#test_getShortClassName_Class
org.apache.commons.lang.ClassUtilsTest#test_getShortClassName_String
org.apache.commons.lang.ClassUtilsTest#test_getPackageName_Object
org.apache.commons.lang.ClassUtilsTest#test_getPackageName_Class
org.apache.commons.lang.ClassUtilsTest#test_getPackageName_String
org.apache.commons.lang.ClassUtilsTest#test_getAllSuperclasses_Class
org.apache.commons.lang.ClassUtilsTest#test_getAllInterfaces_Class
org.apache.commons.lang.ClassUtilsTest#test_convertClassNamesToClasses_List
org.apache.commons.lang.ClassUtilsTest#test_convertClassesToClassNames_List
org.apache.commons.lang.ClassUtilsTest#test_isInnerClass_Class
org.apache.commons.lang.ClassUtilsTest#test_isAssignable_ClassArray_ClassArray
org.apache.commons.lang.ClassUtilsTest#test_isAssignable
org.apache.commons.lang.ClassUtilsTest#test_isAssignable_Widening
org.apache.commons.lang.util.BitFieldTest#testSet
org.apache.commons.lang.util.BitFieldTest#testGetValue
org.apache.commons.lang.util.BitFieldTest#testGetRawValue
org.apache.commons.lang.util.BitFieldTest#testGetShortValue
org.apache.commons.lang.util.BitFieldTest#testGetShortRawValue
org.apache.commons.lang.util.BitFieldTest#testIsSet
org.apache.commons.lang.util.BitFieldTest#testIsAllSet
org.apache.commons.lang.util.BitFieldTest#testSetValue
org.apache.commons.lang.util.BitFieldTest#testSetShortValue
org.apache.commons.lang.util.BitFieldTest#testByte
org.apache.commons.lang.util.BitFieldTest#testClear
org.apache.commons.lang.util.BitFieldTest#testClearShort
org.apache.commons.lang.util.BitFieldTest#testSetShort
org.apache.commons.lang.util.BitFieldTest#testSetBoolean
org.apache.commons.lang.util.BitFieldTest#testSetShortBoolean
org.apache.commons.lang.time.DateUtilsTest#testRound
org.apache.commons.lang.time.DateUtilsTest#testTrunc
org.apache.commons.lang.time.DateUtilsTest#testParse
org.apache.commons.lang.time.DateUtilsTest#testWeekIterator
org.apache.commons.lang.time.DateUtilsTest#testMonthIterator
org.apache.commons.lang.math.LongRangeTest#testIncludesNumber
org.apache.commons.lang.math.LongRangeTest#testIncludesLongBig
org.apache.commons.lang.math.LongRangeTest#testConstructor1a
org.apache.commons.lang.math.LongRangeTest#testConstructor1b
org.apache.commons.lang.math.LongRangeTest#testConstructor2a
org.apache.commons.lang.math.LongRangeTest#testConstructor2b
org.apache.commons.lang.math.LongRangeTest#testEquals
org.apache.commons.lang.math.LongRangeTest#testIncludesRange
org.apache.commons.lang.math.LongRangeTest#testToString
org.apache.commons.lang.math.LongRangeTest#testIncludesInteger
org.apache.commons.lang.math.LongRangeTest#testGetMinimum
org.apache.commons.lang.math.LongRangeTest#testGetMaximum
org.apache.commons.lang.math.LongRangeTest#testIncludesLong
org.apache.commons.lang.math.LongRangeTest#testIncludesDouble
org.apache.commons.lang.math.LongRangeTest#testIncludesFloat
org.apache.commons.lang.math.LongRangeTest#testOverlapsRange
org.apache.commons.lang.math.LongRangeTest#testHashCode
org.apache.commons.lang.StringUtilsTrimEmptyTest#testTrimToNull
org.apache.commons.lang.StringUtilsTrimEmptyTest#testClean
org.apache.commons.lang.StringUtilsTrimEmptyTest#testTrim
org.apache.commons.lang.StringUtilsTrimEmptyTest#testTrimToEmpty
org.apache.commons.lang.StringUtilsTrimEmptyTest#testIsNotEmpty
org.apache.commons.lang.StringUtilsTrimEmptyTest#testIsEmpty
org.apache.commons.lang.StringUtilsTrimEmptyTest#testDeleteSpace
org.apache.commons.lang.StringUtilsTrimEmptyTest#testStrip
org.apache.commons.lang.math.FloatRangeTest#testIncludesNumber
org.apache.commons.lang.math.FloatRangeTest#testToString
org.apache.commons.lang.math.FloatRangeTest#testConstructor1a
org.apache.commons.lang.math.FloatRangeTest#testConstructor1b
org.apache.commons.lang.math.FloatRangeTest#testConstructor2a
org.apache.commons.lang.math.FloatRangeTest#testConstructor2b
org.apache.commons.lang.math.FloatRangeTest#testEquals
org.apache.commons.lang.math.FloatRangeTest#testIncludesRange
org.apache.commons.lang.math.FloatRangeTest#testIncludesInteger
org.apache.commons.lang.math.FloatRangeTest#testGetMinimum
org.apache.commons.lang.math.FloatRangeTest#testGetMaximum
org.apache.commons.lang.math.FloatRangeTest#testIncludesLong
org.apache.commons.lang.math.FloatRangeTest#testIncludesDouble
org.apache.commons.lang.math.FloatRangeTest#testIncludesFloat
org.apache.commons.lang.math.FloatRangeTest#testOverlapsRange
org.apache.commons.lang.math.FloatRangeTest#testHashCode
org.apache.commons.lang.builder.CompareToBuilderTest#testReflectionCompareEx1
org.apache.commons.lang.builder.CompareToBuilderTest#testReflectionHierarchyCompareTransients
org.apache.commons.lang.builder.CompareToBuilderTest#testReflectionCompare
org.apache.commons.lang.builder.CompareToBuilderTest#testReflectionCompareEx2
org.apache.commons.lang.builder.CompareToBuilderTest#testReflectionHierarchyCompare
org.apache.commons.lang.builder.CompareToBuilderTest#testByte
org.apache.commons.lang.builder.CompareToBuilderTest#testAppendSuper
org.apache.commons.lang.builder.CompareToBuilderTest#testObject
org.apache.commons.lang.builder.CompareToBuilderTest#testObjectEx2
org.apache.commons.lang.builder.CompareToBuilderTest#testObjectComparator
org.apache.commons.lang.builder.CompareToBuilderTest#testObjectComparatorNull
org.apache.commons.lang.builder.CompareToBuilderTest#testLong
org.apache.commons.lang.builder.CompareToBuilderTest#testInt
org.apache.commons.lang.builder.CompareToBuilderTest#testShort
org.apache.commons.lang.builder.CompareToBuilderTest#testChar
org.apache.commons.lang.builder.CompareToBuilderTest#testDouble
org.apache.commons.lang.builder.CompareToBuilderTest#testFloat
org.apache.commons.lang.builder.CompareToBuilderTest#testBoolean
org.apache.commons.lang.builder.CompareToBuilderTest#testObjectArray
org.apache.commons.lang.builder.CompareToBuilderTest#testLongArray
org.apache.commons.lang.builder.CompareToBuilderTest#testIntArray
org.apache.commons.lang.builder.CompareToBuilderTest#testShortArray
org.apache.commons.lang.builder.CompareToBuilderTest#testCharArray
org.apache.commons.lang.builder.CompareToBuilderTest#testByteArray
org.apache.commons.lang.builder.CompareToBuilderTest#testDoubleArray
org.apache.commons.lang.builder.CompareToBuilderTest#testFloatArray
org.apache.commons.lang.builder.CompareToBuilderTest#testBooleanArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiLongArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiIntArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiShortArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiCharArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiByteArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiFloatArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiDoubleArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMultiBooleanArray
org.apache.commons.lang.builder.CompareToBuilderTest#testRaggedArray
org.apache.commons.lang.builder.CompareToBuilderTest#testMixedArray
org.apache.commons.lang.builder.CompareToBuilderTest#testObjectArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testLongArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testIntArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testShortArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testCharArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testByteArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testDoubleArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testFloatArrayHiddenByObject
org.apache.commons.lang.builder.CompareToBuilderTest#testBooleanArrayHiddenByObject
org.apache.commons.lang.math.DoubleRangeTest#testIncludesNumber
org.apache.commons.lang.math.DoubleRangeTest#testToString
org.apache.commons.lang.math.DoubleRangeTest#testConstructor1a
org.apache.commons.lang.math.DoubleRangeTest#testConstructor1b
org.apache.commons.lang.math.DoubleRangeTest#testConstructor2a
org.apache.commons.lang.math.DoubleRangeTest#testConstructor2b
org.apache.commons.lang.math.DoubleRangeTest#testEquals
org.apache.commons.lang.math.DoubleRangeTest#testIncludesRange
org.apache.commons.lang.math.DoubleRangeTest#testIncludesInteger
org.apache.commons.lang.math.DoubleRangeTest#testGetMinimum
org.apache.commons.lang.math.DoubleRangeTest#testGetMaximum
org.apache.commons.lang.math.DoubleRangeTest#testIncludesLong
org.apache.commons.lang.math.DoubleRangeTest#testIncludesDouble
org.apache.commons.lang.math.DoubleRangeTest#testIncludesFloat
org.apache.commons.lang.math.DoubleRangeTest#testOverlapsRange
org.apache.commons.lang.math.DoubleRangeTest#testHashCode
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testLongArrayArray
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testAppendSuper
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testObject
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testLong
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testObjectArray
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testLongArray
org.apache.commons.lang.builder.NoFieldNamesToStringStyleTest#testBlank
org.apache.commons.lang.builder.EqualsBuilderTest#testByte
org.apache.commons.lang.builder.EqualsBuilderTest#testObject
org.apache.commons.lang.builder.EqualsBuilderTest#testLong
org.apache.commons.lang.builder.EqualsBuilderTest#testInt
org.apache.commons.lang.builder.EqualsBuilderTest#testShort
org.apache.commons.lang.builder.EqualsBuilderTest#testChar
org.apache.commons.lang.builder.EqualsBuilderTest#testDouble
org.apache.commons.lang.builder.EqualsBuilderTest#testFloat
org.apache.commons.lang.builder.EqualsBuilderTest#testBoolean
org.apache.commons.lang.builder.EqualsBuilderTest#testObjectArray
org.apache.commons.lang.builder.EqualsBuilderTest#testLongArray
org.apache.commons.lang.builder.EqualsBuilderTest#testIntArray
org.apache.commons.lang.builder.EqualsBuilderTest#testShortArray
org.apache.commons.lang.builder.EqualsBuilderTest#testCharArray
org.apache.commons.lang.builder.EqualsBuilderTest#testByteArray
org.apache.commons.lang.builder.EqualsBuilderTest#testDoubleArray
org.apache.commons.lang.builder.EqualsBuilderTest#testFloatArray
org.apache.commons.lang.builder.EqualsBuilderTest#testBooleanArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiLongArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiIntArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiShortArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiCharArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiByteArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiFloatArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiDoubleArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMultiBooleanArray
org.apache.commons.lang.builder.EqualsBuilderTest#testRaggedArray
org.apache.commons.lang.builder.EqualsBuilderTest#testMixedArray
org.apache.commons.lang.builder.EqualsBuilderTest#testObjectArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testLongArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testIntArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testShortArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testCharArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testByteArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testDoubleArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testFloatArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testBooleanArrayHiddenByObject
org.apache.commons.lang.builder.EqualsBuilderTest#testReflectionEquals
org.apache.commons.lang.builder.EqualsBuilderTest#testReflectionHierarchyEquals
org.apache.commons.lang.builder.EqualsBuilderTest#testSuper
org.apache.commons.lang.ObjectUtilsTest#testIdentityToString
org.apache.commons.lang.ObjectUtilsTest#testIsNull
org.apache.commons.lang.ObjectUtilsTest#testNull
org.apache.commons.lang.ObjectUtilsTest#testEquals
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionArrayAndObjectCycle
org.apache.commons.lang.builder.ToStringBuilderTest#testAppendToString
org.apache.commons.lang.builder.ToStringBuilderTest#testIntArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testShortArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testByteArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testCharArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testDoubleArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testFloatArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testBooleanArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testByte
org.apache.commons.lang.builder.ToStringBuilderTest#testLongArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testConstructorEx1
org.apache.commons.lang.builder.ToStringBuilderTest#testConstructorEx2
org.apache.commons.lang.builder.ToStringBuilderTest#testConstructorEx3
org.apache.commons.lang.builder.ToStringBuilderTest#testGetSetDefault
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionInteger
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionCharacter
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionBoolean
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionObjectArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionLongArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionIntArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionShortArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionyteArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionCharArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionDoubleArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionFloatArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionBooleanArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionFloatArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionLongArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionIntArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testSetDefaultEx
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionhortArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionByteArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionCharArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionDoubleArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionBooleanArrayArray
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionHierarchyArrayList
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionHierarchy
org.apache.commons.lang.builder.ToStringBuilderTest#testInnerClassReflection
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionArrayCycle
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionArrayCycleLevel2
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionArrayArrayCycle
org.apache.commons.lang.builder.ToStringBuilderTest#testSimpleReflectionObjectCycle
org.apache.commons.lang.builder.ToStringBuilderTest#testReflectionObjectCycle
org.apache.commons.lang.builder.ToStringBuilderTest#testAppendSuper
org.apache.commons.lang.builder.ToStringBuilderTest#testObject
org.apache.commons.lang.builder.ToStringBuilderTest#testLong
org.apache.commons.lang.builder.ToStringBuilderTest#testInt
org.apache.commons.lang.builder.ToStringBuilderTest#testShort
org.apache.commons.lang.builder.ToStringBuilderTest#testChar
org.apache.commons.lang.builder.ToStringBuilderTest#testDouble
org.apache.commons.lang.builder.ToStringBuilderTest#testFloat
org.apache.commons.lang.builder.ToStringBuilderTest#testBoolean
org.apache.commons.lang.builder.ToStringBuilderTest#testObjectArray
org.apache.commons.lang.builder.ToStringBuilderTest#testLongArray
org.apache.commons.lang.builder.ToStringBuilderTest#testIntArray
org.apache.commons.lang.builder.ToStringBuilderTest#testShortArray
org.apache.commons.lang.builder.ToStringBuilderTest#testCharArray
org.apache.commons.lang.builder.ToStringBuilderTest#testByteArray
org.apache.commons.lang.builder.ToStringBuilderTest#testDoubleArray
org.apache.commons.lang.builder.ToStringBuilderTest#testFloatArray
org.apache.commons.lang.builder.ToStringBuilderTest#testBooleanArray
org.apache.commons.lang.builder.ToStringBuilderTest#testBlank
org.apache.commons.lang.EntitiesTest#testUnescapeUnknownEntity
org.apache.commons.lang.EntitiesTest#testTreeIntMap
org.apache.commons.lang.EntitiesTest#testHashIntMap
org.apache.commons.lang.EntitiesTest#testArrayIntMap
org.apache.commons.lang.EntitiesTest#testBinaryIntMap
org.apache.commons.lang.EntitiesTest#testEscapeNamedEntity
org.apache.commons.lang.EntitiesTest#testUnescapeNamedEntity
org.apache.commons.lang.EntitiesTest#testAddEntitiesArray
org.apache.commons.lang.EntitiesTest#testEntitiesXmlObject
org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest#testFixture
org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest#testInteger
org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest#testIntegerWithTransients
org.apache.commons.lang.builder.HashCodeBuilderAndEqualsBuilderTest#testFixtureWithTransients
org.apache.commons.lang.EntitiesPerformanceTest#testBuildHash
org.apache.commons.lang.EntitiesPerformanceTest#testBuildTree
org.apache.commons.lang.EntitiesPerformanceTest#testBuildArray
org.apache.commons.lang.EntitiesPerformanceTest#testBuildBinary
org.apache.commons.lang.EntitiesPerformanceTest#testLookupHash
org.apache.commons.lang.EntitiesPerformanceTest#testLookupTree
org.apache.commons.lang.EntitiesPerformanceTest#testLookupArray
org.apache.commons.lang.EntitiesPerformanceTest#testLookupBinary
org.apache.commons.lang.EntitiesPerformanceTest#testEscapeHash
org.apache.commons.lang.EntitiesPerformanceTest#testEscapeTree
org.apache.commons.lang.EntitiesPerformanceTest#testEscapeArray
org.apache.commons.lang.EntitiesPerformanceTest#testEscapeBinary
org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeHash
org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeTree
org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeArray
org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeBinary
org.apache.commons.lang.BooleanUtilsTest#test_toStringOnOff_Boolean
org.apache.commons.lang.BooleanUtilsTest#test_toStringYesNo_Boolean
org.apache.commons.lang.BooleanUtilsTest#test_toString_Boolean_String_String_String
org.apache.commons.lang.BooleanUtilsTest#test_toStringTrueFalse_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toStringOnOff_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toStringYesNo_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toString_boolean_String_String_String
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toBoolean_Boolean
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanDefaultIfNull_Boolean_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toBoolean_int
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_int
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_Integer
org.apache.commons.lang.BooleanUtilsTest#test_toBoolean_int_int_int
org.apache.commons.lang.BooleanUtilsTest#test_toBoolean_Integer_Integer_Integer
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_int_int_int
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_Integer_Integer_Integer
org.apache.commons.lang.BooleanUtilsTest#test_toInteger_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toIntegerObject_boolean
org.apache.commons.lang.BooleanUtilsTest#test_toIntegerObject_Boolean
org.apache.commons.lang.BooleanUtilsTest#test_toInteger_boolean_int_int
org.apache.commons.lang.BooleanUtilsTest#test_toInteger_Boolean_int_int_int
org.apache.commons.lang.BooleanUtilsTest#test_toIntegerObject_boolean_Integer_Integer
org.apache.commons.lang.BooleanUtilsTest#test_toIntegerObject_Boolean_Integer_Integer_Integer
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_String
org.apache.commons.lang.BooleanUtilsTest#test_toBooleanObject_String_String_String_String
org.apache.commons.lang.BooleanUtilsTest#test_toBoolean_String
org.apache.commons.lang.BooleanUtilsTest#test_toBoolean_String_String_String
org.apache.commons.lang.BooleanUtilsTest#test_toStringTrueFalse_Boolean
org.apache.commons.lang.BooleanUtilsTest#test_negate_Boolean
org.apache.commons.lang.RandomStringUtilsTest#testRandomStringUtils
org.apache.commons.lang.RandomStringUtilsTest#testRandomAlphaNumeric
org.apache.commons.lang.RandomStringUtilsTest#testRandomNumeric
org.apache.commons.lang.RandomStringUtilsTest#testRandomAscii
org.apache.commons.lang.RandomStringUtilsTest#testRandomAlphabetic
org.apache.commons.lang.RandomStringUtilsTest#testRandomStringUtilsHomog
org.apache.commons.lang.NumberRangeTest#testEquals
org.apache.commons.lang.NumberRangeTest#testMaxMin
org.apache.commons.lang.NumberRangeTest#testIncludesNumber
org.apache.commons.lang.NumberRangeTest#testIncludesRange
org.apache.commons.lang.NumberRangeTest#testConstructorNullParameters
org.apache.commons.lang.NumberRangeTest#testToString
org.apache.commons.lang.builder.StandardToStringStyleTest#testLongArrayArray
org.apache.commons.lang.builder.StandardToStringStyleTest#testAppendSuper
org.apache.commons.lang.builder.StandardToStringStyleTest#testObject
org.apache.commons.lang.builder.StandardToStringStyleTest#testLong
org.apache.commons.lang.builder.StandardToStringStyleTest#testObjectArray
org.apache.commons.lang.builder.StandardToStringStyleTest#testLongArray
org.apache.commons.lang.builder.StandardToStringStyleTest#testBlank
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testLongArrayArray
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testAppendSuper
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testObject
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testLong
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testObjectArray
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testLongArray
org.apache.commons.lang.builder.MultiLineToStringStyleTest#testBlank
org.apache.commons.lang.NullArgumentExceptionTest#testConstructor_nullInput
org.apache.commons.lang.NullArgumentExceptionTest#testGetMessage_nullConstructorInput
org.apache.commons.lang.NullArgumentExceptionTest#testGetMessage_validConstructorInput
org.apache.commons.lang.SerializationUtilsTest#testSerializeStream
org.apache.commons.lang.SerializationUtilsTest#testSerializeStreamUnserializable
org.apache.commons.lang.SerializationUtilsTest#testSerializeStreamNullObj
org.apache.commons.lang.SerializationUtilsTest#testSerializeStreamObjNull
org.apache.commons.lang.SerializationUtilsTest#testSerializeStreamNullNull
org.apache.commons.lang.SerializationUtilsTest#testDeserializeStream
org.apache.commons.lang.SerializationUtilsTest#testDeserializeStreamOfNull
org.apache.commons.lang.SerializationUtilsTest#testDeserializeStreamNull
org.apache.commons.lang.SerializationUtilsTest#testDeserializeStreamBadStream
org.apache.commons.lang.SerializationUtilsTest#testSerializeBytes
org.apache.commons.lang.SerializationUtilsTest#testSerializeBytesUnserializable
org.apache.commons.lang.SerializationUtilsTest#testSerializeBytesNull
org.apache.commons.lang.SerializationUtilsTest#testDeserializeBytes
org.apache.commons.lang.SerializationUtilsTest#testDeserializeBytesOfNull
org.apache.commons.lang.SerializationUtilsTest#testDeserializeBytesNull
org.apache.commons.lang.SerializationUtilsTest#testDeserializeBytesBadStream
org.apache.commons.lang.SerializationUtilsTest#testClone
org.apache.commons.lang.SerializationUtilsTest#testCloneNull
org.apache.commons.lang.SerializationUtilsTest#testCloneUnserializable
org.apache.commons.lang.StringUtilsEqualsIndexOfTest#testEquals
org.apache.commons.lang.StringUtilsEqualsIndexOfTest#testEqualsIgnoreCase
org.apache.commons.lang.StringUtilsEqualsIndexOfTest#testLastIndexOfAny
org.apache.commons.lang.StringUtilsEqualsIndexOfTest#testIndexOfAny
org.apache.commons.lang.math.IntRangeTest#testIncludesNumber
org.apache.commons.lang.math.IntRangeTest#testIncludesIntegerBig
org.apache.commons.lang.math.IntRangeTest#testConstructor1a
org.apache.commons.lang.math.IntRangeTest#testConstructor1b
org.apache.commons.lang.math.IntRangeTest#testConstructor2a
org.apache.commons.lang.math.IntRangeTest#testConstructor2b
org.apache.commons.lang.math.IntRangeTest#testEquals
org.apache.commons.lang.math.IntRangeTest#testIncludesRange
org.apache.commons.lang.math.IntRangeTest#testToString
org.apache.commons.lang.math.IntRangeTest#testIncludesInteger
org.apache.commons.lang.math.IntRangeTest#testGetMinimum
org.apache.commons.lang.math.IntRangeTest#testGetMaximum
org.apache.commons.lang.math.IntRangeTest#testIncludesLong
org.apache.commons.lang.math.IntRangeTest#testIncludesDouble
org.apache.commons.lang.math.IntRangeTest#testIncludesFloat
org.apache.commons.lang.math.IntRangeTest#testOverlapsRange
org.apache.commons.lang.math.IntRangeTest#testHashCode
org.apache.commons.lang.IncompleteArgumentExceptionTest#test1arg_nullInput
org.apache.commons.lang.IncompleteArgumentExceptionTest#test1arg_validInput
org.apache.commons.lang.IncompleteArgumentExceptionTest#test2arg_allNullInput
org.apache.commons.lang.IncompleteArgumentExceptionTest#test2arg_nullString
org.apache.commons.lang.IncompleteArgumentExceptionTest#test2arg_nullArray
org.apache.commons.lang.IncompleteArgumentExceptionTest#test2arg_validInput
org.apache.commons.lang.WordWrapUtilsTest#testWrapText3
org.apache.commons.lang.WordWrapUtilsTest#testWrapText1
org.apache.commons.lang.WordWrapUtilsTest#testWrapText2
```

Reply:
Tests that should've been executed but were not:
```
>>> baseline - actual
{'org.apache.commons.lang.EntitiesPerformanceTest#testEscapeHash', 
'org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeTree', 
'org.apache.commons.lang.SystemUtilsTest#testOSMatches', 
'org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeHash', 
'org.apache.commons.lang.EntitiesPerformanceTest#testEscapeTree', 
'org.apache.commons.lang.EntitiesPerformanceTest#testUnescapeBinary', 
'org.apache.commons.lang.SystemUtilsTest#testJavaVersionAsFloat', 
'org.apache.commons.lang.NumberUtilsTest#testConstants', 
'org.apache.commons.lang.SystemUtilsTest#testJavaVersionAsInt', 
'org.apache.commons.lang.SystemUtilsTest#testJavaVersionMatches', 
'org.apache.commons.lang.SystemUtilsTest#testOSMatches2', 
'org.apache.commons.lang.EntitiesPerformanceTest#testLookupBinary', 
'org.apache.commons.lang.SerializationUtilsTest#testDeserializeBytesNull', 
'org.apache.commons.lang.enums.EnumTest#testOperationCalculation'}
```

Tests that where executed but shouldn't have been:
```
>>> actual - baseline
{'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateGetThrowables', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testGetAccessibleMethod', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetMessages', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetMessages', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetMessage', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetThrowableI', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testSimpleStatic3', 
'org.apache.commons.lang.reflect.ReflectionUtilsTestCase#testWidening', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testInvokeMethod', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateGetThrowableCount', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testPrintPartialStackTrace', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testIndexOfThrowable', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetMessageI', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetCause', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetMessage', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateGetMessages', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetMessageI', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testInvokeMethodWithPrimitives', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetMessageI', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testPrintStackTrace', 
'org.apache.commons.lang.exception.ExceptionUtilsTestCase#testPrintThrowables', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetThrowableI', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetThrowableCount', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateGetMessageN', 
'org.apache.commons.lang.exception.ExceptionUtilsTestCase#testGetCause', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetThrowables', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetThrowableCount', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testPrintStackTrace', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testIndexOfThrowableI', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetThrowableCount', 
'org.apache.commons.lang.exception.ExceptionUtilsTestCase#testGetRootCause', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testIndexOfThrowableI', 
'org.apache.commons.lang.reflect.ReflectionUtilsTestCase#testScopeUtils', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testPrintPartialStackTrace', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testIndexOfThrowable', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateGetThrowableN', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testPrintPartialStackTrace', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateConstructor', 
'org.apache.commons.lang.exception.ExceptionUtilsTestCase#testIsNestedThrowable', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testPublicSub', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testinvokeMethod2', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegetePrintStackTrace', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetThrowableI', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testSimpleStatic2', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testIndexOfThrowableI', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testGetThrowables', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetThrowables', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetMessage', 
'org.apache.commons.lang.reflect.MethodUtilsTestCase#testSimpleStatic1', 
'org.apache.commons.lang.exception.ExceptionUtilsTestCase#testGetThrowableCount', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetMessages', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testIndexOfThrowable', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testPrintStackTrace', 
'org.apache.commons.lang.exception.NestableErrorTestCase#testIndexOfThrowable', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testSerialization', 
'org.apache.commons.lang.exception.NestableDelegateTestCase#testNestableDelegateGetMessage', 
'org.apache.commons.lang.exception.NestableExceptionTestCase#testGetCause', 
'org.apache.commons.lang.exception.NestableRuntimeExceptionTestCase#testGetCause'}
```